src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -6,6 +6,11 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
```

Comment:
ℹ Microsoft.NETFramework.ReferenceAssemblies is implicitly referenced when needed starting with SDK 5.0.100, so we can remove this if require that minimum SDK version on all platforms.